student.rb

```diff
@@ -4,7 +4,7 @@ class Student < Person
   attr_accessor :classroom
 
   def initialize(age, classroom, name = 'Unknown', parent_permission: true)
```

Comment:
The user is not able to create a student due to an error. Kindly see the error below.

![student](https://user-images.githubusercontent.com/35504149/189356631-05655ba8-345a-4ebc-bb6c-3250ac493735.png)

Reply:
Great i have worked on it